health/notifications/hangouts/README.md

```diff
@@ -1,23 +1,33 @@
 <!--
----
-title: "Hangouts Chat"
+title: "Send notifications to Google Hangouts"
+description: "Send alerts to Send notifications to Google Hangouts any time an anomaly or performance issue strikes a node in your infrastructure."
+sidebar_label: "Google Hangouts"
 custom_edit_url: https://github.com/netdata/netdata/edit/master/health/notifications/hangouts/README.md
----
 -->
 
-# Hangouts Chat
+# Send notifications to Google Hangouts
+
+[Google Hangouts](https://hangouts.google.com/) is cross-platform messaging app developed by Google. You can configure
+Netdata to send alarm notifications to a Hangouts room in order to stay aware of possible health or performance issues
+on your nodes. Here's an example of the notification in action:
 
-This is what you will get:
 ![Netdata on Hangouts](https://user-images.githubusercontent.com/1153921/66427166-47de6900-e9c8-11e9-8322-b4b03f084dc1.png)
+
 To receive notifications in Google Hangouts, you need the following in your Hangouts setup:
 
-1. One or more rooms
-2. An **incoming webhook** for each room
+1.   One or more rooms.
+2.   An **incoming webhook** for each room.
+
+Follow [Google's documentation](https://developers.google.com/hangouts/chat/how-tos/webhooks) to create an incoming
+webhook for each room you want to send Netdata notifications to.
 
-How to create an incoming webhook: 
-https://developers.google.com/hangouts/chat/how-tos/webhooks
+Set the webhook URIs and room names in `health_alarm_notify.conf`. To edit it on your system, run
+`/etc/netdata/edit-config health_alarm_notify.conf`):
 
-Set the webhook URIs and room names in `health_alarm_notify.conf`. To edit it on your system, run `/etc/netdata/edit-config health_alarm_notify.conf`):
+## Threads (optional)
+
+Instead to receive alarms on different threads, Netdata allows you to concentrate them inside an unique thread when you
```

Comment:
Instead `of receiving`
`a` unique

Reply:
@joelhans you reviewed yesterday, please, can you confirm these @vlvkobal suggestions?

Reply:
I think Joel knows better than me 😉